docs/tutorial.rst

```diff
@@ -6,7 +6,30 @@ During GSOC 2021 a bunch of new tutorials were programmed by Hrithik Kumar Verma
 a `public share <https://fz-juelich.sciebo.de/s/IzNGi24Ki68iV7t#pdfviewer>`_
 
 
-Some videos about the Mission Support System (MSS)
+Get familiar by some videos about the Mission Support System (MSS).
+
+
+  .. video:: _static/mp4/tutorial_waypoints.mp4
+     :nocontrols:
```

Comment:
Does this mean that there are no controls shown to skip around in the videos? I think that would not be good (and was also not great with the gifs), being able to skip around is useful.

Reply:
I change that, the autoplay is here the more important. I want one tutorial on top of the page.